pgModel/models.go

```diff
@@ -0,0 +1,65 @@
+package pgModel
+
+type JwtToken struct {
+    Token string `json:"token"`
+}
+
+type Exception struct {
+    Message string `json:"message"`
+}
+
+type User struct {
+	Id     int64 `sql:",pk,unique"`
+	Username   string
+	Email string `sql:",unique,notnull"`
+	Password string
+	PhoneNumber string
+	Messages []*Message
+	Connections []*Connection
+	Comments []*Comment
+	Collections []*Collection
+}
+
+type Connection struct {
+	Id int64
+	FirstUserId int64
+	SecondUserId int64
+}
+
+type Message struct {
+	Id int64
+	UserId int64
+	Content string
+	MessageSender string
+    MessageReceiver string
+}
+
+type Resource struct {
+   Id int64
+   CollectionId int64
+   Link string
+   Type string
+   Views int64
+   Recommendations int64
+   Comments []*Comment
```

Comment:
@mikey2020 I see inconsistent indentations in the struct definitions

Reply:
Indentations dont really matter when creating structs